t/Net/DRI/Protocol/EPP/Extensions/EURid.t

```diff
@@ -33,11 +33,11 @@ my ($rc,$s,$d,$co,$toc,$cs,$h,$dh,@c);
 ########################################################################################################
 ## Process greetings to select namespace versions
 # This greeting was taken off release_notes_march_2019
```

Comment:
Can you remove/correct this note please? Its misleading now :)
(note more more below. No need to change comments on all domain examples as only namespace has changed, but it feels relevant in the greeting)

Reply:
Makes sense. Comments removed/updated.